### PR TITLE
tokeninfo does not require both access_token and id_token

### PR DESCRIPTION
**Pull request by robertdolca**
We can't use only id_token because access_token is needed in other
areas.

### CODE REVIEW

File: spec/omniauth/strategies/google_oauth2_spec.rb [hunk 1]
```diff
@@ -565,16 +565,16 @@
 
     it 'should verify token if access_token and id_token are valid and app_id equals' do
```
Comment (zquestz) [inline spec/omniauth/strategies/google_oauth2_spec.rb]: We should update the test names to reflect the new functionality.

**Comment by zquestz** [comment]
This looks good once the test names are updated. Thanks.

**Comment by robertdolca** [comment]
Fixed

**Comment by robertdolca** [comment]
@zquestz clould you please make a new release with this fix?

**Comment by zquestz** [comment]
Released.

**Pull request merged**